core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -38,7 +38,8 @@ <h4 tabindex="0" [ngClass]="(checkpointCount === 1) ? 'd-none' : 'progress-bar-l
           </div>
           <div *ngFor="let checkpointNode of checkpointStatusArray; index as idx"
                [ngClass]="{'completed-checkpoint-node': checkpointStatusArray[idx] === 'completed', 'in-progress-checkpoint-node': checkpointStatusArray[idx] === 'in-progress', 'incomplete-checkpoint-node': checkpointStatusArray[idx] === 'incomplete'}">
-            {{ idx + 1 }}
+            <a *ngIf="checkpointStatusArray[idx] === 'completed'" (click)="validateIndexAndGoToCheckpoint(idx)">{{ idx + 1 }}</a>
```

Comment:
Anchor tags without an href cause accessibility issues for keyboard users, wouldn't a button be better suited for this? Or we could have `href="#"` perhaps.